Source/WebCore/svg/SVGPatternElement.cpp

```diff
@@ -157,31 +157,31 @@ RenderPtr<RenderElement> SVGPatternElement::createElementRenderer(RenderStyle&&
 
 void SVGPatternElement::collectPatternAttributes(PatternAttributes& attributes) const
 {
-    if (!attributes.hasX() && hasAttribute(SVGNames::xAttr))
+    if (!attributes.hasX() && (isAnimatedAttribute(SVGNames::xAttr) || hasAttribute(SVGNames::xAttr)))
```

Comment:
Hmm, wouldn't a helper method does check both be more convenient?

Reply:
Yes - I will get back to this - I think I need to introduce helper function because `isAnimateAttributes` lead to failures and with using both `||` still leads to failures. Let me rebase this first and push and then do local testing and fix this.